setup.py

```diff
@@ -1,19 +1,32 @@
 """Install nix-bisect"""
 
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(
     name="nix-bisect",
     version="0.4.1",
     description="Bisect nix builds",
     author="Timo Kaufmann",
-    packages=["nix_bisect"],
+    include_package_data=True,
+    packages=find_packages(),
+    py_modules = [
+        'bisect_env',
+        'bisect_runner',
+        'build_status',
+        'derivation',
+        'exceptions',
+        'extra_bisect',
+        'gcroot',
+        'git',
+        'git_bisect',
+        'nix',
+        'test_util',
+    ],
     install_requires=["appdirs", "pexpect",],
-    entry_points={
-        "console_scripts": [
-            "nix-build-status=nix_bisect.build_status:_main",
-            "bisect-env=nix_bisect.bisect_env:_main",
-            "extra-bisect=nix_bisect.extra_bisect:_main",
-        ]
-    },
+    entry_points='''
+        [console_scripts]
+        nix-build-status=nix_bisect.build_status:_main
+        bisect-env=nix_bisect.bisect_env:_main
+        extra-bisect=nix_bisect.extra_bisect:_main
+    ''',
```

Comment:
The dict-based argument should work. Did you encounter any errors with the old configuration?

Reply:
Yes it worked, I messed up this, sorry. Fixed.